core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java

```diff
@@ -354,6 +355,17 @@ public Response getWebUIOverview() {
         }
       }
 
+      Gauge masterRoleIdGauge = MetricsSystem.METRIC_REGISTRY.getGauges()
+          .get(MetricKey.MASTER_ROLE_ID.getName());
+      Gauge leaderIdGauge = MetricsSystem.METRIC_REGISTRY.getGauges()
+          .get(MetricKey.CLUSTER_LEADER_ID.getName());
```

Comment:
When are they `null`? If they are null does it mean there's a bug somewhere and we should log a warning?

Reply:
When these metrics haven't register to MetricsSystem